docs/package_templates/prebuilt_tool_package/all/conanfile.py

```diff
@@ -0,0 +1,61 @@
+from conan import ConanFile
+from conan.tools.files import get, copy
+import os
+
+
+required_conan_version = ">=1.47.0"
+
+
+class PackageConan(ConanFile):
+    name = "package"
+    description = "short description"
+    license = "" # Use short name only, conform to SPDX License List: https://spdx.org/licenses/
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/project/package"
+    topics = ("topic1", "topic2", "topic3", "pre-built") # no "conan"  and project name in topics
```

Comment:
```suggestion
    topics = ("topic1", "topic2", "topic3") # no "conan"  and project name in topics
```

Is this a topic we want to have in all the pre-built packages? If not, I would just keep the _regular ones_.

Reply:
we may have one topic, why not?

Reply:
Yes, I would love having a standard for topics, like `header-only` mandatory for any header only recipe and `pre-built` for tooling. So we could easily filter it on Conan Center web site.

Reply:
ok, just wanted to know this was intentional